Set default flag to 2 for TCP traceflow (#4948)
antrea-io/antrea: docs/traceflow-guide.md

      tcp:
        srcPort: 10000 # Source port needs to be set when Protocol is TCP/UDP.
        dstPort: 80 # Destination port needs to be set when Protocol is TCP/UDP.

>         flags: 2 # Construct a SYN packet, when Protocol is TCP, flags will be set to 2 by default when it's not provided.

Reviewer: ```suggestion
        flags: 2 # Construct a SYN packet: 2 is also the default value when the flags field is omitted.
```

Author: Updated.
